components/brave_wallet/browser/wallet_data_files_installer.cc

```diff
@@ -24,8 +24,8 @@
 
 #if BUILDFLAG(BRAVE_WALLET_ENABLED)
```

Comment:
You don't need this since this file won't be built if it is not enabled.

Reply:
Fixed in 00a17f6f17500a72e33cd0760ab2e297b9db06ec